kanidm_client/tests/default_entries.rs

```diff
@@ -0,0 +1,622 @@
+#![deny(warnings)]
+use std::collections::HashSet;
+
+use kanidm_client::KanidmClient;
+use kanidm_proto::v1::{Filter, Modify, ModifyList};
+
+mod common;
+use crate::common::{run_test, ADMIN_TEST_PASSWORD};
+
+static USER_READABLE_ATTRS: [&str; 9] = [
```

Comment:
I wonder if these should be in #cfg(test) in constants near the definitions of the access controls so that they are updated in sync?

Reply:
Where? I don't know where you're referring sorry :S

Reply:
if you move the USER_READABLE_ATTRS to kanidmd/src/lib/constants/, then mark the module as `#[cfg(test)]`, so that way we can have USER_READABLE_ATTRS defined near the access controls?

Reply:
WIP